ContactsManager/ContactsManager_ContactsManager_interface.html

```diff
@@ -0,0 +1,103 @@
+<div>{{draft}}{{securecontext_header}}{{DefaultAPISidebar("Contact Picker API")}}</div>
+
+<p class="summary">The <strong><code>ContactsManager</code></strong> interface of the {{domxref('Contact Picker API')}} allows users to select entries from their contact list and share limited details of the selected entries with a website or application.</p>
+
+<p>The <code>ContactsManager</code> is available through the global {{domxref('navigator.contacts')}} property and must be called on user gesture.</p>
```

Comment:
This would be better mentioned on the select method page, as this is the method that actually requires user gesture. The landing page explains it, so I've moved it from here.